understanding/20/use-of-color.html

```diff
@@ -8,6 +8,15 @@
 <body>
    <h1>Understanding Use of Color</h1>
    
+   <section id="brief">
+		<h2>In brief</h2>
+		<dl>
+		<dt>Objective</dt><dd>Color is not the only way of distinguishing information</dd>
+		<dt>Author task</dt><dd>Use other things than just color to convey meaning</dd>
+		<dt>Key beneficiaries</dt><dd>Blind users and those with low vision</dd>
```

Comment:
@gundulaniemann said:
> users benefiting most are color-blind users.
Only some of them might be low vision as well (in the legal sense).
Blind users do not benefit from visuals (unless they see but are legally blind).
Suggestion:
Color-blind users and those with reduced sight

As per prior comments, de-emphasizing the Key beneficiaries, but agree blind users are not key, so have matched Contrast (Minimum)

```suggestion
		<dt>Key beneficiaries</dt><dd>Users with low vision</dd>
```